Components/Hlms/Pbs/src/Cubemaps/OgreCubemapProbe.cpp

```diff
@@ -380,6 +388,23 @@ namespace Ogre
         if( workspaceDefOverride != IdString() )
             workspaceDef = compositorManager->getWorkspaceDefinition( workspaceDefOverride );
 
+        const CompositorWorkspaceDef::NodeAliasMap& workspaceNodes = workspaceDef->getNodeAliasMap();
+        assert( workspaceNodes.size() == 1 );
+
+        CompositorNodeDef* probeNode = compositorManager->getNodeDefinitionNonConst( workspaceNodes.begin()->first );
+        assert( probeNode->getNumTargetPasses() % 6 == 0 );
```

Comment:
Can I assume that the node is composed by six (or multiple of it) target passes?

Reply:
Assumption made: node must be composed by `6 * k + 1 ` target passes (`6 * k` cubemap _face passes_ plus an additional ibl_specular pass).